services/121-service/test/visa-card/block-visa-card.test.ts

```diff
@@ -73,43 +78,51 @@ describe('(Un)Block visa debit card', () => {
 
   it('should succesfully unblock a Visa Debit card', async () => {
     // Arrange
-    await seedPaidRegistrations([registrationVisa], programIdVisa);
+    const testRegistration = {
+      ...registrationVisa,
+      referenceId: 'test-registration-visa--unblock-card',
+    };
+    await seedPaidRegistrations([testRegistration], programIdVisa);
 
-    // Act
-    const visaWalletResponseBeforeBlock = await getVisaWalletsAndDetails(
+    const visaWalletResponseBefore = await getVisaWalletsAndDetails(
       programIdVisa,
-      registrationVisa.referenceId,
+      testRegistration.referenceId,
       accessToken,
     );
-    const tokencode = visaWalletResponseBeforeBlock.body.cards[0].tokenCode;
+    const tokencode = visaWalletResponseBefore.body.cards[0].tokenCode;
 
     await blockVisaCard(
       programIdVisa,
       tokencode,
       accessToken,
-      registrationVisa.referenceId,
+      testRegistration.referenceId,
     );
+
+    // Act
     const unblockVisaResponse = await unblockVisaCard(
       programIdVisa,
       tokencode,
       accessToken,
-      registrationVisa.referenceId,
+      testRegistration.referenceId,
     );
-    const visaWalletResponse = await getVisaWalletsAndDetails(
+
+    // Assert
+    await waitFor(4_000); // the last message otherwise was not in the db yet
```

Comment:
@elwinschmitz You could instead use this helper function here 

    await waitForMessagesToComplete({
      programId: programIdOCW,
      referenceIds: [referenceIds[0]],
      accessToken,
      minimumNumberOfMessagesPerReferenceId: 2, // not sure what this needs to be in your case..
    });

Reply:
Ah. yes. I only looked at the file as-is;

But there is indeed a need for some "overview of available helper-methods" :)

I've created a generic 'bucket' backlog-item for those kind of refactors: AB#36871